docs/apple_app_signing.md

```diff
@@ -39,12 +39,14 @@ To use a GitHub Actions to build and sign your app:
 
 1. Send your GitHub repo's name to the [Developer Experience team](contact.md). 
 1. We will give it access to the following secrets:
-  1. `BUILD_CERTIFICATE_BASE64`
-  1. `IOS_BUILD_CERTIFICATE_PASSWD`
+  1. `APPLE_APP_STORE_BUILD_CERTIFICATE_BASE64`
+  1. `APPLE_APP_STORE_BUILD_CERTIFICATE_PASSWD`
 1. Create and download the provisioning profile for your app
 1. Install the provisioning profile as a secret in your repo
   1. Refer to GitHub's [Installing an Apple certificate on macOS runners for Xcode development](https://docs.github.com/en/actions/deployment/deploying-xcode-applications/installing-an-apple-certificate-on-macos-runners-for-xcode-development) documentation for details
-1. In the Signing & Capabilities -> Release tab in Xcode turn off "automatically manage signing" 
+1. In the Signing & Capabilities -> Release tab in Xcode:
```

Comment:
Replace ampersands with the full word "and" when possible. Ampersands can be challenging for the visually impaired, non-English speakers and those using assistive technologies to distinguish.

Reply:
Xcode labels the tab as "Signing & Capabilities" (with the &). I copied the label from Xcode as that is what is displayed to the developer. I can change it though if it doesn't need to match

Reply:
Leaving as is, as it is how Xcode labels it